client/src/components/ThreadForm/ThreadForm.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Button from '../Button/Button';
+
+const ThreadForm = ({textAreaProps, ...props}) => {
```

Comment:
Isn't the name 'ThreadForm' misleading? We still have to implement the "thread" part.
I think, "message" or something like this would be more appropriate.

Reply:
`message` is a very generic name, which means different things in different contexts. The name `ThreadForm` is set because it is for the Thread page and used for the specific purpose of sending messages on discussion (aka Threads). Also, there will be some changes made at later stages like adding an emoji, or @mentioning someone, which are related to a discussion thread.

Reply:
@rakeshtembhurne, any thoughts on this?